content/docs/ref/comment.md

```diff
@@ -63,11 +70,32 @@ Any [generic option](/doc/ref) in addition to:
 
 ## Examples
 
+### Post a text report
+
+Write to your report using whatever method you prefer. For example, copy the
+contents of a text file containing the results of ML model training:
+
+```cli
+$ cat results.txt >> report.md
+```
+
+### Post a graphic report
+
+Display images using Markdown or HTML. Note that if an image is an output of
```

Comment:
FYI this was moved here from the current https://cml.dev/doc/usage#cml-reports (hard to find/ somewhat unrelated there).